build.sbt

```diff
@@ -55,37 +57,44 @@ lazy val `coordination-etcd` =
       libraryDependencies ++= Seq(
         library.akkaHttp,
         library.circeParser,
-        library.akkaTestkit % Test,
-        library.scalaTest   % Test
+        library.akkaTestkit           % Test,
+        library.scalaTest             % Test,
+        library.dockerTestKit         % Test,
+        library.dockerTestKitSpotify  % Test
       )
     )
 
+parallelExecution in Global := false
```

Comment:
The problem was with inter-submodules concurrency. Usually it is that multi nodes tests are run before `EtcdCoordinationSpec`'s `afterAll` was called. It is not desired because than creating etcd container fails because one is still running (and has not been stopped yet).

It was the easiest fix for that. I am open for any suggestions.